Makefile

```diff
@@ -67,13 +71,11 @@ install.tools: .install.gitvalidation .install.glide .install.glide-vc .install.
 	gometalinter --install --update
 
 clean:
-	rm -rf *~ $(OUTPUT_DIRNAME)
-	rm -f oci-create-runtime-bundle
-	rm -f oci-unpack
-	rm -f oci-image-validate
+	rm -rf *~ $(OUTPUT_DIRNAME) $(TOOLS)
 
 .PHONY: \
 	tools \
+	$(TOOLS) \
```

Comment:
also, not true. When those binaries are built from the subdirectory, then the executable by those names is present.